### PR TITLE
Example for `Carbonite.insert_transaction`

### PR DESCRIPTION
**Pull request by andreasknoepfle**
I guess there is some folks around that don't like `Ecto.Multi`. So we might want to add an example for `Carbonite.insert_transaction` as well. Wdyt @maltoe ?

### CODE REVIEW

**Comment by andreasknoepfle** [comment]
See #66

File: README.md [hunk 1]
```diff
@@ -247,6 +247,20 @@ Ecto.Multi.new()
 
 As you can see, the `Carbonite.Transaction` is a great place to store metadata for the operation. A `type` field can be used to categorize the transactions. A `user_id` would be a good candidate for a transaction log, as well.
 
+### Without Ecto.Multi 
+
+If you have a situation, where you want to insert a transaction without `Ecto.Multi`, then you can use `Carbonite.insert_transation/3`. Although grouping multiple operations into one transaction fits the use-case of `Ecto.Multi` perfectly so `Carbonite.Multi.insert_transaction/2` is still the preferred way here.
```
Comment (maltoe) [inline README.md:252]:
```suggestion
If you have a situation where you want to insert a transaction without `Ecto.Multi`, you can use `Carbonite.insert_transation/3`.
```

IMO this sentence about the multi being the preferred way: :shrug: it's mentioned first, guess that's enough?

File: README.md [hunk 2]
```diff
@@ -247,6 +247,20 @@ Ecto.Multi.new()
...
+``` elixir
+MyApp.Repo.transaction(fn ->
+  {:ok, _} = Carbonite.insert_transaction(MyApp.Repo, %{meta: %{type: "rabbit_inserted"}})
+
+  params
+  |> MyApp.Rabbit.create_changeset()
+  |> MyApp.Repo.insert!
+end)
+```
+
```
Comment (maltoe) [inline README.md:263]: How about removing the following section about the manual manual way of inserting a transaction through `Transaction.changeset/1`?

**Review by maltoe — COMMENTED** [review]
Good change :star2:

File: README.md (hunk 2 above)
Comment (andreasknoepfle) [reply README.md:263]: Not sure, I think that is still nice to have. In the end there might be cases for this as well, who knows...

Comment (maltoe) [reply README.md:263]: OK, but then maybe let's remove the 

> If you don't have the luxury of an Ecto.Multi

below. Starting sentences of these two sections now make it appear that they don't know of each other

File: README.md [hunk 3]
```diff
@@ -247,6 +247,20 @@ Ecto.Multi.new()
...
 ### Building a changeset for manual insertion
 
 If you don't have the luxury of an `Ecto.Multi`, you can create a changeset for a `Carbonite.Transaction` using `Carbonite.Transaction.changeset/1`:
```
Comment (andreasknoepfle) [inline README.md:266]:
```suggestion
Another alternative is creating a changeset for a `Carbonite.Transaction` using `Carbonite.Transaction.changeset/1`:
```

**Comment by andreasknoepfle** [comment]
Done @maltoe

**Review by maltoe — APPROVED** [review]

**Pull request merged**